dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -618,6 +625,66 @@ public Token addBitbucketToken(@ApiParam(hidden = true) @Auth User user, @QueryP
         }
     }
 
+    @POST
+    @Timed
+    @UnitOfWork
+    @Path("/orcid.org")
+    @ApiOperation(value = "Add a new orcid.org token", authorizations = {
+            @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "This is used as part of the OAuth 2 web flow. "
+            + "Once a user has approved permissions for ORCID, "
+            + "their browser will load the redirect URI which should resolve here", response = Token.class)
+    @Operation(operationId = "addOrcidToken", summary = "Request and store tokens from ORCID API", security = @SecurityRequirement(name = "bearer"))
```

Comment:
Make sure that the openapi.yaml changes that are checked-in roughly matches the swagger.yaml changes

Reply:
Looks about the same to me